index.html

```diff
@@ -26,11 +26,29 @@ <h1 class="newsletter-title">Vue.js Newsletter</h1>
           </div>
         </form>
       </div>
-      <div class="newsletter__previous"><a href="/issues.html">Browse list of previous issues</a></div>
+
+      <div class="content">
+        <div class="content-read">
+            <p class="content-title">Read the News</p>
+            <div class="newsletter__previous"><a href="/issues.html">Browse previous issues</a></div>
+          </div>
+    
+          <div class="content-listen">
+            <p class="content-title">Listen to the Podcast</p>
+            <a class="podcast-subscribe" href="https://itunes.apple.com/us/podcast/the-official-vue-news/id1329151772" target="_blank">Apple Podcasts</a>
+            <a class="podcast-subscribe" href="https://subscribeonandroid.com/vuenews.fireside.fm/rss" target="_blank">Android</a>
+            <a class="podcast-subscribe" href="https://vuenews.fireside.fm/rss" target="_blank">RSS</a>            
+          </div>
+      </div>
+
+      <p class="footer">
+          Created by <a href="https://twitter.com/DamianDulisz" target="_blank">@damiandulisz</a>, <a href="https://twitter.com/greggpollack" target="_blank">@greggpollack</a>, <a href="https://twitter.com/adamjahr" target="_blank">@adamjahr</a> and <a href="https://twitter.com/venticco" target="_blank">@venticco</a>.
+          <br>
+          Proudly supported by <a href="http://monterail.com" target="_blank"><img src="monterail-logo.png" class="monterail-link"/>@monterail</a> and <a href="http://monterail.com" target="_blank"> <img src="vuemastery-logo.png" class="vuemastery-logo"/>@VueMastery</a>.
```

Comment:
VueMastery link is still pointing to `http://monterail.com` address.

Reply:
Also the image `vuemastery-logo.png` file is missing in the commit.